.checkstyle/checkstyle.xml

```diff
@@ -87,7 +87,10 @@
         <!-- code quality -->
         <module name="MethodLength"/>
         <module name="ParameterNumber"/>
-        <module name="ClassDataAbstractionCoupling"/>
+        <module name="ClassDataAbstractionCoupling">
+            <!-- default is 7 -->
+            <property name="max" value="8"/>
+        </module>
```

Comment:
Would annotation be better then overriding it here? It will make sure you don't change it everywhere but only where really needed.